README.md

```diff
@@ -134,15 +134,17 @@ That process is a little tricky, but here's one way to do it: assuming this code
 
     qvm-run --pass-io sd-dev 'tar -c -C /home/user/projects securedrop-workstation' | tar xvf -
 
+(Be sure to include the space after the parent directory.)
 After that initial manual step, the code in your development VM may be copied into place on `dom0` by setting the `SECUREDROP_DEV_VM` and `SECUREDROP_DEV_DIR` environmental variables to reflect the VM and directory to which you've cloned this repo, and running `make clone` from the root of the project on `dom0`:
 
 ```
-export SECUREDROP_DEV_VM=sd-dev    # set to your dev VM
-export SECUREDROP_DEV_DIR=/home/user/projects/securedrop-workstation    # set to your working directory
-make clone
+[dom0]$ export SECUREDROP_DEV_VM=sd-dev    # set to your dev VM
+[dom0]$ export SECUREDROP_DEV_DIR=/home/user/projects/securedrop-workstation    # set to your working directory
+[dom0]$ cd projects/securedrop-workstation
```

Comment:
There's a note in the README that the dom0 home for the code must be /home/user/securedrop-workstation. Not sure if it's still a requirement, but this command would contradict that.